### PR TITLE
Promote internationalization by default

### PR DESCRIPTION
**Pull request by DrewAPicture**
I've stubbed out an initial pass at adding default internationalization recommendations as suggested in #114. Work in progress.

### CODE REVIEW

File: _includes/markdown/PHP.md [hunk 1]
```diff
@@ -616,6 +616,29 @@ if ( ! empty( $_POST['_wpnonce'] ) && wp_verify_nonce( $_POST['_wpnonce'], 'my_a
...
+* Internationalized output should always be escaped
+
+##### Text Domains
+
+Each project should leverage a unique text domain for its strings. Text domains should be lowercase, alphanumeric, and use hyphens to separate multiple words: `tenup-project-name`.
+
+Text domains should never be stored in a variable or constant when used with core localization functions, as this practice can often produce unexpected results.
+
+{Stub}
+If the code is for release as a plugin or theme in the WordPress.org repositories, the text domain *must* match the directory slug for the project in order to ensure compatibility with the WordPress language pack delivery system. he text domain should be defined in the "Text Domain" header in the plugin or stylesheet headers, respectively.
```
Comment (stevegrunwell) [inline _includes/markdown/PHP.md:634]: Typo: "he text domain" should be "**The** text domain".

**Comment by stevegrunwell** [comment]
It may be worth including a link to @otto42's [_Internationalization: You're Probably Doing it Wrong_](http://ottopress.com/2012/internationalization-youre-probably-doing-it-wrong/) post here, as I've had to point people there before as a nice way of saying "don't get too clever."

**Comment by tlovett1** [comment]
Great start, @DrewAPicture! Excited to see this finished.

**Comment by tlovett1** [comment]
@DrewAPicture ping :)

**Comment by dana-ross** [comment]
@DrewAPicture second ping :) I'd love to see this finished up & merged. Once PHP+WordPress internationalization is formally part of our best practices, we can start thinking about the best way to approach it in the JavaScript frameworks we adopt.

**Comment by DrewAPicture** [comment]
@daveross Yep, top of my list for this week.

**Comment by RachelRVasquez** [comment]
@tlovett1 Saw the help wanted label on this and wanted to see where I can help? 😄 I'm still solidifying my understanding of internationalization and saw this as a good opportunity to strengthen that understanding. Since a PR was already started, should I copy the changes Drew had already started and then just continue from there and submit a PR of my own?

**Comment by tlovett1** [comment]
Would love to see this carried over the finish line.

**Comment by tlovett1** [comment]
Anyone?

**Comment by dana-ross** [comment]
@tlovett1 I'll submit a new pull request

**Pull request merged**